packages/aws-cdk/lib/init-templates/app/typescript/package.template.json

```diff
@@ -1,24 +1,20 @@
 {
-    "name": "%name%",
-    "version": "0.1.0",
-    "main": "bin/index.js",
-    "types": "bin/index.d.ts",
-    "bin": {
-        "%name%": "bin/%name%.js"
-    },
-    "scripts": {
-        "build": "tsc",
-        "watch": "tsc -w",
-        "cdk": "cdk"
-    },
-    "devDependencies": {
-        "@types/node": "^8.9.4",
-        "typescript": "^3.1.2",
-        "aws-cdk": "^%cdk-version%"
-    },
-    "dependencies": {
-        "@aws-cdk/aws-sns": "^%cdk-version%",
-        "@aws-cdk/aws-sqs": "^%cdk-version%",
-        "@aws-cdk/cdk": "^%cdk-version%"
-    }
+  "name": "%name%",
+  "version": "0.1.0",
+  "bin": {
+    "%name%": "bin/%name%.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@types/node": "^8.9.4",
+    "typescript": "^3.1.2",
+    "aws-cdk": "^%cdk-version%"
+  },
+  "dependencies": {
+    "@aws-cdk/cdk": "^%cdk-version%"
+  }
```

Comment:
Add `peerDependencies` on `@aws-cdk/cdk`

Reply:
This is an app. It doesn't need peerDependencies.